Source/WebCore/rendering/RenderTable.cpp

```diff
@@ -1179,7 +1179,7 @@ LayoutUnit RenderTable::calcBorderStart() const
         if (sectionAdjoiningBorder.style() > BorderStyle::Hidden)
             borderWidth = std::max(borderWidth, sectionAdjoiningBorder.width());
 
-        if (const RenderTableCell* adjoiningStartCell = topNonEmptySection->firstRowCellAdjoiningTableStart()) {
+        if (const RenderTableCell* adjoiningStartCell = topNonEmptySection->cellAt(0, 0).primaryCell()) {
             // FIXME: Make this work with perpendicular and flipped cells.
```

Comment:
is this (and the one below) change relevant?

Reply:
I was trying to figure that out too, so I just left it in there from the original patch. I haven't really touched this code before, so someone else would probably have a better idea. 

I am ultimately not sure what this line was trying to solve here. Why did we need to cap the Column index at 0 if we do not have the same direction.
```cpp
unsigned adjoiningEndCellColumnIndex = isDirectionSame(this, table()) ? table()->lastColumnIndex() : 0;
```